### ISSUE TITLE
[TalkBack] Touch exploration and speech interfere with Play Music lock screen widget

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#186]
```
What steps will reproduce the problem?
1. play music using google playmusic app 
2. lock screen but with screen on 
3. talkback/midi tones interfer with music progress 

What is the expected output? What do you see instead?

there should be no interference on the lock screen widgets. talkback focus is 
misplaced with beta2

What version of the product are you using? On what operating system?
4.2.2

Please provide any additional information below.


```

Original issue reported on code.google.com by `redger...@gmail.com` on 24 Feb 2013 at 10:55

**Comment by GoogleCodeExporter** [comment]
```
Unable to reproduce interference with music progress, but while the Play Music 
lock screen widget is showing there are some serious graphical corruption 
issues when TalkBack is speaking and focus management issues when touch 
exploring.
```

Original comment by `alanv@google.com` on 6 Mar 2013 at 1:45
- Changed title: **[TalkBack] Touch exploration and speech interfere with Play Music lock screen widget**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
Original comment by `alanv@google.com` on 6 Mar 2013 at 1:46

**Comment by GoogleCodeExporter** [comment]
Original comment by `alanv@google.com` on 8 Mar 2013 at 7:25
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `alanv@google.com` on 29 May 2013 at 7:53

**Comment by GoogleCodeExporter** [comment]
```
I have same problems with the lock screen of Nexus 10.
```

Original comment by `electrog...@gmail.com` on 24 Jun 2013 at 7:46

**Comment by GoogleCodeExporter** [comment]
```
Same issue on Nexus five
```

Original comment by `blindst...@gmail.com` on 5 Apr 2014 at 8:28
